packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -110,7 +128,48 @@ export async function startConsolidatedQuery({
   return progress;
 }
 
-async function getConsolidatedAndSendToCx(params: GetConsolidatedParams): Promise<void> {
+function appendProgressToProcessingQueries(
+  currentConsolidatedQueries: ConsolidatedQuery[] | undefined,
+  progress: ConsolidatedQuery
+): ConsolidatedQuery[] {
+  if (currentConsolidatedQueries) {
+    const queriesInProgress = currentConsolidatedQueries.filter(
+      query => query.status === "processing"
```

Comment:
Every time we run the consolidated query we want to clear out any progresses that were already completed or failed

Reply:
Yea to confirm its only to display the progress to cxs we dont use this for any webhook logic